test/transformers/test_rms_norm.py

```diff
@@ -183,3 +183,96 @@ def test_correctness_functional(bs, sl, hd, dtype, atol, rtol, reference, offset
     y2.backward(grad)
 
     assert torch.allclose(h1.grad, h2.grad, atol=atol, rtol=rtol)
+
+
+def estimate_mem(bs, sl, hd, dtype):
+    """
+    Estimate GPU memory requirement for the test case.
+    """
+    # Get bytes per element for the dtype
+    if dtype == torch.float32:
+        bytes_per_element = 4
+    elif dtype == torch.bfloat16:
+        bytes_per_element = 2
```

Comment:
automate the process
```suggestion
    bytes_per_element = torch.tensor([], dtype=dtype).element_size()
```